e2eshark/run.py

```diff
@@ -1344,6 +1349,16 @@ def main():
         action="store_true",
         default=False,
     )
+    parser.add_argument(
```

Comment:
Let us be consistent with types. Since they are floating point numbers, can we simply require it to take a floating point value with no default? Then you check if args.rtol then use it, else use the default set in getTolerances function.
For each of atol and rtol:

parser.add_argument(
        "--atol",
        help="Set atol tolerance value",
        type=float,
    )

parser.add_argument(
        "--rtol",
        help="Set rtol tolerance value",
        type=float,
    )

Reply:
ok changed it ; will push in couple of mins